README.md

```diff
@@ -1,2 +1,30 @@
-# studying-ts-automation-process
-My own repo, my own tests
+**Project**: studying-ts-automation-process
+
+**Project setup**
+- Visual Studio code
+- Node.js
+- TypeScript
+- Cypress
+//will add here some new points later
+
+**How to run the project**
+- Create a new folder for your project on your local PC
+- Cloned the repository from Github https://github.com/AlexandraPopovaz/studying-ts-automation-process.git
+- Run **git init** (create your local repo)
+- Run **npm init** (create a Package.json file)
+- Run **npm install cypress --save-dev** (install Cypress app - two folders will be created: 
```

Comment:
Redundant, if you know about `npm` then you should know about `node_modules` and lock-file. Please add not common information, but related to your project.

Reply:
Fixed